src/Controller/ProgrammeController.php

```diff
@@ -69,4 +75,34 @@ public function showExactSearchByName(Request $request): Response
 
         return new JsonResponse($serializedData, Response::HTTP_OK, [], true);
     }
+
+    /**
+     * @Route(path="/sort/participants", methods={"GET"})
```

Comment:
Exposing this method has the following issues:
- your API is no longer RESTful
- sorting should be possible on the main GET route for programs (through an optional query parameter)